src/debug.rs

```diff
@@ -26,8 +26,6 @@ impl<T> DebugView<T> {
 /// ```
 /// extern crate tree_sitter;
 /// extern crate tree_sitter_powershell;
```

Comment:
Why not also deleting the follwing extern crate in the test comments? 
```diff
- extern crate tree_sitter;
- extern crate tree_sitter_powershell;
```

Reply:
I tried at first, but removing those would cause the doctests to fail.

Reply:
I see, those modules don't require it because they inherit the declaration from `src/lib.rs`. I'll remove the declarations in the `ps` modules.